tests/integration/security/testdata/authz/v1beta1-conditions.yaml.tmpl

```diff
@@ -42,14 +42,20 @@ spec:
     when:
     - key: request.headers[x-foo]
       values: ["foo"]
+  - to:
+    - operation:
+        paths: ["/request-headers-regex"]
+    when:
+    - key: request.headers[x-foo-regex]
+      values: ["regex:foo-regexp[0-9]?"]
 ---
 
 # Each of the following authorization policy uses a different condition on the given path.
 
 apiVersion: security.istio.io/v1beta1
 kind: AuthorizationPolicy
 metadata:
-  name: condition-request-headers-notvalues
+  name: condition-request-headers
```

Comment:
why change the name here? It's duplicate of the other policy and will override it.